### DISCUSSION TITLE
Welcome to hotBath Discussions!

### DISCUSSION THREAD

**Discussion by crabsatellite** [crabsatellite/hotBath#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LokiOdinsohn** [comment]
Hello and I wish you a good day. I apologize in advance for my bad English. Please forgive me for that. Well, further into the text, I have an idea. You could also bring in a bucket of hot slime. We can bring it in some anime like this Example Overlord can be seen and you could also combine it with different herbs for different effects and maybe a few board-like building blocks like stone, iron or something like that, which you could then place in a block of deep water as a seat for in the water and while you're at it why don't you make another mod for steam baths 

 kind regards 
Loki
